### PR TITLE
Create testing for API library 

### PR DESCRIPTION
**Pull request by connorguy**

### CODE REVIEW

File: test/test_documents_api.py [hunk 1]
```diff
@@ -34,57 +35,144 @@ def test_assign_document(self):
...
+        body = {
+            "id": 46530,
+            "email": "user@email.com",
+            "is_translator": True,
+            "is_reviewer": False,
+            "due_date": "2019-10-16T22:12:34.000Z",
+        }
+
+        assert doc_api.assign_document(body) == {"id": 46530}
+        assert mock_client.call_api.call_count == 1
```
Comment (connorguy) [inline test/test_documents_api.py]: @tgallant do these tests look sufficient?

Comment (connorguy) [inline test/test_documents_api.py]: @tgallant do these tests look sufficient?

File: test/test_documents_api.py [hunk 2]
```diff
@@ -35,57 +35,144 @@ def test_assign_document(self):
...
+
+        body = {
+            "id": 46530,
+            "email": "user@email.com",
+            "is_translator": True,
+            "is_reviewer": False,
+            "due_date": "2019-10-16T22:12:34.000Z",
+        }
+
+        assert doc_api.assign_document(body) == {"id": 46530}
```
Comment (tgallant) [inline test/test_documents_api.py]: On second thought I think we may want to mock out a different method for these types of tests. The return value of these api methods should be an object with a specific type, not a dict. It looks like that `call_api` method handles the deserialization as well so we shouldn't mock that. There is actually a `request` method [here](https://github.com/lilt/lilt-python/blob/master/lilt/api_client.py#L369) that just handles the http call.

I think we could update all of these to something like

```python
from unittest.mock import Mock, patch

...

    @patch.object(lilt.ApiClient, 'request')
    def test_assign_document(self, mock_request):
        """Test case for assign_document

        Assign a Document  # noqa: E501
        """
        mock_response = Mock()
        mock_response.data = json.dumps({"id": 46530})
        mock_request.return_value = mock_response
        api_client = lilt.ApiClient()
        doc_api = DocumentsApi(api_client)

        body = {
            "id": 46530,
            "email": "user@email.com",
            "is_translator": True,
            "is_reviewer": False,
            "due_date": "2019-10-16T22:12:34.000Z",
        }

        assignment_response = doc_api.assign_document(body)
        assert assignment_response.id == 46530
```

I think the main goal of these tests should be to test the deserialization from the json api response to the appropriate data type.

**Review by tgallant — APPROVED** [review]
Looking good to me. I left a comment inline but good to go after that

**Pull request merged**